dashboard/config/foorm/forms/surveys/pd/ayw_workshop_pre_survey.0.json

```diff
@@ -0,0 +1,1369 @@
+{
+ "pages": [
+  {
+   "name": "page 1",
+   "elements": [
+    {
+     "type": "html",
+     "name": "pre_survey_preamble",
+     "baker_comment":"Whatever this becomes we should put into the workshop_elements library and pull from there",
+     "html": "<h2>{workshop_course} Workshop Pre-Survey</h2><br><p>NEED TO FILL IN PREAMBLE CONTENT</p>"
+    },
+
+    {
+     "type": "radiogroup",
+     "name": "workshop_course",
+     "title": "For which Code.org course are you attending a workshop?",
+     "baker_comment":"Molly, will this survey have workshop_course available?",
+     "choices": [
+      {
+       "value": "CS Principles",
+       "text": "CS Principles"
+      },
+      {
+       "value": "CS Discoveries",
+       "text": "CS Discoveries"
+      }
+     ]
+    },
+    {
+     "type": "radiogroup",
+     "name": "is_virtual",
+     "title": "Is the workshop you're attending in-person, or virtual?",
+     "choices": [
+      {
+       "value": "false",
+       "text": "In-Person"
+      },
+      {
+       "value": "true",
+       "text": "Virtual"
+      }
+     ]
+    },
+    {
+     "type": "radiogroup",
+     "name": "planning_to_attend",
+     "title": " Are you planning to attend this upcoming workshop?",
+     "choices": [
+      {
+       "value": "yes",
+       "text": "Yes"
+      },
+      {
+       "value": "no",
+       "text": "No"
+      },
+      {
+       "value": "unsure",
+       "text": "Unsure / Don't Know Yet"
+      }
+     ]
+    },
+    {
+     "type": "panel",
+     "name": "csp_most_recent_unit_panel",
+     "elements": [
+      {
+       "type": "dropdown",
+       "name": "csp_unit",
+       "visibleIf": "{workshop_course} = 'CS Principles'",
+       "title": "In which unit from CS Principles is the lesson you most recently taught?",
```

Comment:
possible clearer phrasing for this question: "Which unit from CS Principles contains the lesson you most recently taught?"

Reply:
I like it!  Changing.